### ISSUE TITLE
Four Parshios in pyluach

### ISSUE THREAD

**Issue by gershonbaron** [simlist/pyluach#31]
It seems there is no support for Four Parshiot (Shekalim, Zachor, Para, Hodesh) in pyluach. Is that correct?

**Comment by simlist** [comment]
That is correct. I would consider adding it though. Maybe along with any special Maftir.

**Comment by simlist** [comment]
Added in v2.3.0. See [https://pyluach.readthedocs.io/en/latest/parshios.html#pyluach.parshios.four_parshios](https://pyluach.readthedocs.io/en/latest/parshios.html#pyluach.parshios.four_parshios).
